src/components/DataSession/OperationPipeline.vue

```diff
@@ -36,6 +44,40 @@ function operationBtnColor(index) {
 	}
 }
 
+async function pollOperationCompletion(operationID) {
+	const url = await store.state.datalabApiBaseUrl + 'datasessions/' + props.data.id + '/operations/' + operationID + '/'
+
+	const response = await fetchApiCall({ url: url, method: 'GET', failCallback: handleError })
+
+	if (response && response.percent_completion !== undefined && response.status != 'COMPLETED') {
```

Comment:
I think this can be the successCallback. e.g.:
```
returnOperationPercentage(data) {
if (response && response.percent_completion !== undefined && response.status != 'COMPLETED) {
etc...
}
```

Reply:
Agreed.